packages/cli/generators/controller/templates/src/controllers/controller-rest-template.ts.ejs

```diff
@@ -18,46 +18,46 @@ export class <%= name %>Controller {
     public <%= repositoryNameCamel %> : <%= repositoryName %>,
   ) {}
 
-  @post('/<%= modelNameCamel %>')
+  @post('/<%= modelNamePluralKebab %>')
```

Comment:
Our variable names should reflect the intent (what is the URL path), not the implementation details (a plural+kebab version of the name). Let's rename this variable to something like `modelUrlPath` please.